posts/2022-06-24-rust-unconference.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Announcing The RustConf PostConf UnConf"
+author: Jane Lusby, on behalf of The Rust Project Teams
+---
+
+Hello Rust community!
+
+We're excited to announce that the Rust project teams be hosting an
+unconference[^1] the day after RustConf.
+
+![RustConf PostConf UnConf promo image](../../../images/2022-06-24-rust-unconference/Eventbrite_Unconf.jpg)
+
+The PostConf Unconf will be dedicated to the Rust project and will be a
+fantastic opportunity for users, contributors, and maintainers to network and
+discuss the project's development.
+
+There will be no set agenda; instead, attendees will decide what will be
+discussed together and can move freely between sessions to find ones in which
+they can contribute most effectively based on their individual interests and
+needs.
+
+To get the most out of the unconference, jot down your thoughts ahead of time
+and bring them ready to share. We will also setup a channel in the RustConf
+Discord for folks to communicate and make preliminary, informal plans.
+
+If you plan to attend, please register as soon as possible to help us plan
```

Comment:
I expect we'll get questions about remote attendance. Should we address that here?

Reply:
That seems like a good idea. I'm not sure how we'd do remote attendance for a free form event like this, wdyt?

Reply:
That's a good question... It would be helpful to have some sort of conferencing equipment available for use but that might not be possible at such an event.

Reply:
I want to make sure we have projectors and are setup so people can give presentations if they want, but I don't anticipate wanting to run a livestream. Recording on the other hand seems reasonably low effort and high pay off if we end up having any presentations.

Reply:
For now I think I'd like to leave this as is and follow up with a FAQ to answer this and any other questions that we encounter.